platform/feature-api/api/pod-api/src/api.ts

```diff
@@ -175,28 +175,34 @@ export interface Info {
 }
 
 /**
- * `Network` specifies how features can communicate with other devices or servers.
+ * @hidden
+ */
+export interface PolyLifecycle {
+    listFeatures(): Promise<Record<string, boolean>>;
+    startFeature(id: string, background: boolean): Promise<void>;
+}
+
+/**
+ * `Endpoint` is the API features communicate with in order to perform fetch requests
  */
-export interface Network {
+export interface Endpoint {
     /**
-     * A way for features to send HTTP POST requests
-     *
-     * @returns an error message if something went wrong, `undefined` upon success.
+     * Perform a http post request via the endpoint in the pod
+     * @param body the necessary content of the call
+     * @returns a promise with the response
      */
-    httpPost(
-        url: string,
-        body: string,
+    send(
+        endpointId: string,
+        payload: string,
```

Comment:
As long as you're changing these names, maybe also change `authorization` to something a bit more specific? Like `authToken`?